tests/test_model.py

```diff
@@ -19,3 +23,27 @@ def test_querying(create, tmpdir):
     assert len(set(db.todos())) == 9
     assert len(set(db.todos(lists='ab'))) == 6
     assert len(set(db.todos(lists='ab', location='a'))) == 2
+
+
+def test_retain_tz(tmpdir, runner, create, default_database):
+    create(
+        'ar.ics',
+        'SUMMARY:blah.ar\n'
+        'DUE;VALUE=DATE-TIME;TZID=ART:20160102T000000\n'
```

Comment:
'America/Argentina/Buenos_Aires' should work, does Argentina really have that many different timezones today? I count 13, in the database, or is it just for historical purposes?